contracts/hermez/lib/InstantWithdrawManager.sol

```diff
@@ -225,9 +191,15 @@ contract InstantWithdrawManager is HermezHelpers {
      */
     function safeMode() external onlyGovernance {
         // all buckets to 0
-        for (uint256 i = 0; i < _NUM_BUCKETS; i++) {
-            buckets[i] = Bucket(0, 0, 0, 0, 0);
-        }
+        nBuckets = 1;
+        buckets[0] = packBucket(
+            0xFFFFFFFF_FFFFFFFF_FFFFFFFF,
+            0,
+            0,
+            1,
+            0,
+            0
+        );
```

Comment:
could be better to set nBuckets to 0?

And if not, could be better change `0xFFFFFFFF_FFFFFFFF_FFFFFFFF` for 0 instead?

Reply:
No buckets -> No limits.